openedx/adg/lms/branding_extension/tests/helpers_tests.py

```diff
@@ -49,3 +56,23 @@ def test_get_copyright(mocker):
     )
 
     assert get_copyright() == '\u00A9 {} {}'.format(2021, test_platform_name)
+
+
+def test_is_referred_by_login_or_register_with_no_referred_value():
+    request = RequestFactory().get(reverse('root'))
+    flag = is_referred_by_login_or_register(request)
+    assert flag is False
+
+
+@pytest.mark.parametrize(
+    'path, expected_value',
+    [('login', True), ('register', True), ('dashboard', False)],
+    ids=['redirect_from_login', 'redirect_from_register', 'redirect_from_dashboard']
+)
+def test_is_referred_by_login_or_register(path, expected_value):
+    request = RequestFactory().get(reverse('root'))
```

Comment:
Do we really need to get root here? How about just creating request with RequestFactory(), adding meta and testing the method?